### PR TITLE
Add preserve filter

### PR DESCRIPTION
**Pull request by SHooZ**

### CODE REVIEW

File: hamlpy/parser/filters.py [hunk 1]
```diff
@@ -37,6 +37,10 @@ def plain(content, indent, options):
     return textwrap.dedent(content)
 
 
+def preserve(content, intent, options):
+    return textwrap.dedent(content.replace('\n', '&#10;').replace('\r', '&#13;'))
```
Comment (rowanseymour) [inline hamlpy/parser/filters.py]: This should match https://github.com/haml/haml/blob/master/lib/haml/helpers.rb#L132

Comment (rowanseymour) [inline hamlpy/parser/filters.py]: This should match https://github.com/haml/haml/blob/master/lib/haml/helpers.rb#L132

**Comment by SHooZ** [comment]
Error with :highlight filter is caused by my commit?

File: hamlpy/parser/filters.py [hunk 2]
```diff
@@ -37,6 +37,13 @@ def plain(content, indent, options):
     return textwrap.dedent(content)
 
 
+def preserve(content, intent, options):
```
Comment (rowanseymour) [inline hamlpy/parser/filters.py]: typo, intent -> indent

Comment (rowanseymour) [inline hamlpy/parser/filters.py]: typo, intent -> indent

**Comment by rowanseymour** [comment]
@SHooZ looks good, thanks!

**Comment by rowanseymour** [comment]
Made new release with this https://pypi.python.org/pypi/django-hamlpy/1.0.1

**Pull request merged**